package.json

```diff
@@ -72,8 +72,10 @@
     "postinstall": "husky install"
   },
   "devDependencies": {
+    "@testing-library/dom": "^8.17.1",
```

Comment:
Do we need a direct dependency on this? I didn't see any direct imports from it. `@testing-library/react` depends on it, and I think these queries can be imported. from `@testing-library/react`.

Reply:
Just tried again and confirmed that I get an error:

```
Cannot find module '@testing-library/dom/dist/helpers.js' from '/Users/roberts/Documents/source_new/terra-ui/.yarn/__virtual__/@testing-library-user-event-virtual-3c4cd93763/0/cache/@testing-library-user-event-npm-14.4.3-2d1a75355f-852c48ea6d.zip/node_modules/@testing-library/user-event/dist/cjs/utils/misc'